packages/labs/observers/src/test/intersection_controller_test.ts

```diff
@@ -468,4 +475,38 @@ const canTest = () => {
     await intersectionComplete();
     assert.isTrue(el.observerValue);
   });
+
+  test.skip('IntersectionController<T> type-checks', async () => {
```

Comment:
Why are these type tests skipped?

Reply:
I skipped them as they are type only tests which are tested via the TS compiler successfully building this file. However I can run them as well, although there are no runtime assertions.

Reply:
Fixed